nifi-assembly/NOTICE

```diff
@@ -909,6 +909,11 @@ The following binary components are provided under the Apache Software License v
             Expert Group and released to the public domain, as explained at
             http://creativecommons.org/publicdomain/zero/1.0/
 
+  (ASLv2) LittleProxy
```

Comment:
@ijokarumawak This should not need to be added to the NOTICE, as it is a test dependency only.